.drone.yml

```diff
@@ -55,16 +56,18 @@ pipeline:
     environment:
       - BROWSER=chrome #chrome or firefox
       - SELENIUM_HOST=selenium
-      - SRV_HOST_NAME=owncloud
-      - SRV_HOST_PORT=80
-      - SKELETON_DIR=/var/www/owncloud/tests/acceptance/webUISkeleton
       - SELENIUM_PORT=4444
+      - TEST_SERVER_URL=http://owncloud
+      - TEST_SERVER_FED_URL=http://owncloud
```

Comment:
Note: this can be removed when the core PR that makes the next refactoring of run.sh is merged.